_drafts/2024-05-30-lessons-learned-from-the-latest-aviation-project-announcement.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Lessons Learned from the Latest Aviation Project"
+categories: [Open Mic Announcement, Open Mic]
+---
+Content: The Open mic session starts on Friday, May 31, 2024 at [this link](https://meet.jit.si/open-mic-kbss). The speaker is [Miroslav Blaško](https://kbss.felk.cvut.cz/web/team#miroslav-blaško), leader of the KBSS research group. He will discuss the lessons learned from the latest aviation project, which involves developing an information system for AERO Vodochody AEROSPACE a.s (AVA). This system aims to collect and process operational data to enhance the safety and reliability of aircraft. A key objective of the operational data is to provide input for failure analysis using the Fault Tree Analysis (FTA) technique.
+
+##### The abstract
```

Comment:
Abstract (to be consistent with other posts)

Reply:
Sorry, we do not have any guidelines about that, so I am not sure what feels inconsistent.

Would something like this be more acceptable to you ?:

-----

The development of an information system for AERO Vodochody AEROSPACE a.s. introduced several new challenges related to development workflow and the use of various tools. This talk will describe the development workflow and the use of KBSS tools such as SPipes, SForms, and JOPA. It will cover issues related to project management with GitHub, design with Figma, service orchestration with Docker Compose, and integration of user management with Keycloak. Additionally, it will address challenges related refactoring RDF data and shaping RDF data within REST APIs. The session will conclude with insights into the use of conversational AI in the development process.

Reply:
I just meant to write "Abstract" instead of "The abstract" :-D